README.md

```diff
@@ -18,13 +18,12 @@ Guaranteed supported APIs include:
 
 The future may bring:
 
-*   Authentication
 *   PubSub
 *   TMI/chat
 
 These will not be considered:
 
-*   Twitch kraken API
+*   [Twitch Kraken API](https://github.com/dustinlakin/twitch-rb)
```

Comment:
If  you want to linkify this in the documentation I'd suggest this page instead: https://dev.twitch.tv/docs/v5
And we could add another link to that gem as an alternative

Reply:
I see this way as laconic. I mean: "We don't want to implement Kraken API, but if you want use it — use this gem". Do you still think that separate text mention about gem would be better?

Reply:
Yeah I see it that way now, it can be left.